Gordon360/Models/CCT/Housing/Preference.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("ApplicationID", "Preference1")]
+[Table("Preference", Schema = "Housing")]
+public partial class Preference
```

Comment:
If this is solely a mapping table, it should really be named with both objects that it connects. i.e. `ApplicationPreference` if that is what the mapping is. What is the purpose of this table? What is a `preference`? I would have thought it was a preferred hall or something similar.

Reply:
This is not a mapping table, but I probably do need to add some as you mentioned below. However, this might be a lower priority in my opinion. I very much want to add the mapping tables, but we are running out of time. I plan to fix other things first, and if we have time, I will work on adding them.